lib/lamby/handler.rb

```diff
@@ -68,16 +68,19 @@ def body64
     private
 
     def rack
-      @rack ||= case @options[:rack]
-      when :rest, :api
-        Lamby::RackRest.new @event, @context
-      when :alb
-        Lamby::RackAlb.new @event, @context
-      else
-        Lamby::RackHttp.new @event, @context
+      return @rack if defined?(@rack)
+      @rack = begin
+        type = rack_option
+        klass = Lamby::Rack.lookup type, @event
+        klass ? klass.new(@event, @context) : false
       end
     end
 
+    def rack_option
+      return if ENV['LAMBY_TEST_DYNAMIC_HANDLER']
+      @options[:rack]
+    end
```

Comment:
In order to test that all tests pass when no rack option is passed I return nothing if the `LAMBY_TEST_DYNAMIC_HANDLER` is set. This way we can run the entire test suite again with ignored options. See `./bin/_test` where we do this.